docs/openhabian-DEBUG.md

```diff
@@ -81,9 +82,12 @@ Remember to always let `openhabian-config` update itself on start.
 
 If you want to change anything to work around some not yet fixed bug, you can directly edit the files in and below `/opt/openhabian` on your box. Just do not let openhabian-config update itself on start as that would overwrite your changes.
 
+You can also clone (download) a different openHABian version than the most current one, e.g. if someone offers or asks you to testdrive a development version.
```

Comment:
Probably better said as "if a maintainer / contributor to openHABian offers or asks you to testdrive a development version."

Reply:
ok